chassis_encap_test.go

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2020 eBay Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package goovn
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+
+func TestEncaps(t *testing.T) {
+	db = getOVNDB()
+	if db == dbNB {
+		t.Skip("Skip running chassis test against nb db")
+	}
+	t.Logf("Adding Chassis to OVN SB DB")
+	ocmd, err := ovndbapi.ChassisAdd(CHASSIS_NAME, CHASSIS_HOSTNAME, ENCAP_TYPES, IP, nil, nil,nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = ovndbapi.Execute(ocmd)
+	if err != nil {
+		t.Fatalf("Adding Chassis to OVN failed with err %v", err)
+	}
+	t.Logf("Adding Chassis to OVN Done")
+
+	t.Logf("Gettting Chassis by hostname")
+	chassis, err := ovndbapi.ChassisGet(CHASSIS_HOSTNAME)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(chassis) != 1 {
+		t.Fatalf("err getting chassis, total:%v", len(chassis))
+	}
+	chName := chassis[0].Name
+	t.Logf("Chassis found:%+v", chName)
+
+	// Verify if encaps are created
```

Comment:
Need to verify if encaps are referenced in Chassis's encaps column. You defined it in the Chassis type.

Reply:
ok I'll add that UT too